.github/actions/spelling/expect.txt

```diff
@@ -11,20 +11,13 @@ EOL
 EULA
 Egyw
 GDPR
-Hsl
```

Comment:
These entries are [no longer necessary](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/203#issuecomment-1293919581).

Reply:
This makes me so happy! 🎉